Update examples to storybook 6.4.0-rc.0

IanVS opened the pull request:
This updates our example apps to storybook 6.4.0-rc-0 (includes a fix for correct story sorting!), and verifies that #139 is fixed.

eirslett commented:
@shilman is it possible to push back the SSE -> Websocket migration in Storybook until 6.5? That could buy us some time to figure out how to use Websockets both in Vite and in Storybook, without conflict?

nVitius commented:
Some systems don't allow you to open a process on ports < 1024 if you don't have root permissions. I think the port needs to be configurable.

joshwooding commented:
A related change: https://github.com/storybookjs/storybook/pull/16665

IanVS commented:
I've removed the changes to the config, and just changed this to use the recent release candidate version of storybook, which seems to work out-of-the-box.
